operator/apis/machinelearning.seldon.io/v1/prepack.go

```diff
@@ -76,54 +82,64 @@ type PredictorProtocolsConfig struct {
 }
 
 var (
-	ControllerConfigMapName       = "seldon-config"
-	envSklearnServerImageRelated  = os.Getenv(EnvSklearnServerImageRelated)
-	envXgboostServerImageRelated  = os.Getenv(EnvXgboostserverImageRelated)
-	envMlflowServerImageRelated   = os.Getenv(EnvMlflowserverImageRelated)
-	envTfserverServerImageRelated = os.Getenv(EnvTensorflowImageRelated)
-	envTfproxyServerImageRelated  = os.Getenv(EnvTfproxyImageRelated)
-	relatedImageConfig            = map[string]PredictorServerConfig{}
+	ControllerConfigMapName = "seldon-config"
+	envSklearnServer        = os.Getenv(EnvRelatedImageSklearnServer)
+	envXGBoostServer        = os.Getenv(EnvRelatedImageXGBoostServer)
+	envMLFlowServer         = os.Getenv(EnvRelatedImageMLFlowServer)
+	envSklearnServerV2      = os.Getenv(EnvRelatedImageSklearnServerV2)
+	envXGBoostServerV2      = os.Getenv(EnvRelatedImageXGBoostServerV2)
+	envMLFlowServerV2       = os.Getenv(EnvRelatedImageMLFlowServerV2)
+	envTFServerServer       = os.Getenv(EnvRelatedImageTensorflow)
+	envTFProxyServer        = os.Getenv(EnvRelatedImageTFProxy)
+	relatedImageConfig      = map[string]PredictorServerConfig{}
 )
 
 func init() {
-	if envSklearnServerImageRelated != "" {
-		relatedImageConfig[PrepackSklearnName] = PredictorServerConfig{
+	relatedImageConfig = getRelatedImageConfig()
```

Comment:
We now populated `relatedImageConfig` fully with values read from environmental variables. 
Later logic instead of relying on key in `relatedImageConfig` being not set will rather look at `PredictorImageConfig.ContainerImage` and if it is not set to empty string.